doc/manual/src/language/syntax.md

```diff
@@ -503,6 +503,23 @@ three kinds of patterns:
     > [ 23 {} ]
     > ```
 
+    The complete function argument denoted as an `@`-pattern can be accessed inside default values.
```

Comment:
"denotes" has a specific meaning (semantics) that is not what we want here. We're just talking about syntax and binding. So let's use words/phrease like "brought in scope" "bound over" etc.

Here's a shot at that:
```suggestion
    The variable bound by the `@`-pattern is also brought into scope over the set pattern on the other side of the `@`, not just over the body of the function.
    It can therefore be used default values:
```

Reply:
```suggestion
    The full function argument bound by the `@`-pattern is also brought into scope over the set pattern on the other side of the `@`, not just over the body of the function.
    It can therefore be used in default values:
```